cloud/services/virtualmachines/virtualmachines.go

```diff
@@ -75,6 +75,11 @@ func (s *Service) Reconcile(ctx context.Context, spec interface{}) error {
 		return err
 	}
 
+	plan := generateImagePlan(*vmSpec)
+	if err != nil {
```

Comment:
Not required (since there isnt a err anymore), you could probably use it directly below

```
Plan:     generateImagePlan(*vmSpec),
```